distribution/docs/src/main/resources/content/_configuring/configuring-idp.adoc

```diff
@@ -31,14 +30,14 @@ Install the included IdP implementation.
 .. Disable the *Limit RelayStates to 80 Bytes* option to allow interoperability with Service Providers that are not compliant with the SAML Specifications and send RelayStates larger than 80 bytes.
 . Configure Guest Access:
 .. Disable the *Allow Guest Access* option to disallow a user to authenticate against the IdP with a guest account.
-. Configure SP Metadata:
-.. Select the `+` next to SP Metadata to add a new entry.
+. Configure the Service Providers Metadata:
+.. Select the `+` next to *SP Metadata* to add a new entry.
 .. Populate the new entry with:
-... an HTTPS URL (https://),
+... an HTTPS URL (https://) e.g. https://localhost:8993/services/saml/sso/metadata1,
```

Comment:
✏️ I know it was already `e.g.` in the previous version, but would you mind changing it to `, such as`